pkg/api/v1/types.go

```diff
@@ -1023,6 +1023,54 @@ type ReplicationControllerList struct {
 	Items []ReplicationController `json:"items" description:"list of replication controllers; see http://releases.k8s.io/HEAD/docs/replication-controller.md"`
 }
 
+// DaemonSpec is the specification of a daemon.
+type DaemonSpec struct {
```

Comment:
Since pkg/expapi is not yet ready, an alternative would be to leave these types where they are for now, but not expose them in master yet. I think that will be sufficient for them to not show up in documentation. When pkg/expapi is ready, we'd move them.

cc @uluyol